public/profiles/default.js

```diff
@@ -2,6 +2,8 @@
 var configProfile = {
   /* general items */
 
+  googleTagManagerID: 'GTM-WPXWRDV',
```

Comment:
Hi @LucaFumis one question. Is the default profile going to be used for the Netlify deploys and also when in development mode (`yarn start`)? If so, I think the `googleTagManagerID` should be `null` so that we don't send data that is not representative of actual users to google analytics. I'm thinking there should be another profile for the public production deployment and a default profile for development and Netlify deployments.

Reply:
Thank you @mirandaio - good point. There were also other issues, so all things considered we decided with @andrewhercules to not include these changes for now, so won't be merging the PR.